code/WorkInProgress/autopsy.dm

```diff
@@ -162,14 +162,7 @@
 
 	if(istype(usr,/mob/living/carbon))
```

Comment:
These are not questions you can ask about shitcode. It doesn't care about systems, or helpers, or literally any other line of code in the codebase. It does things its way.